utils/testrunner/Matchers.js

```diff
@@ -43,50 +43,60 @@ class Expect {
 
     function applyMatcher(matcherName, matcher, inverse, value, ...args) {
       const result = matcher.call(null, value, ...args);
-      const message = result.message || `expect.${matcherName} failed`;
+      const message = `expect.${matcherName} failed` + (result.message ? `: ${result.message}` : '');
       console.assert(result.pass !== inverse, message);
     }
   }
 }
 
 const DefaultMatchers = {
   toBe: function(value, other, message) {
```

Comment:
If you want to be fancy
function (value, other, message=`${value} == ${other}`)

Reply:
nice! I'll try it out some other time